app/_locales/en/messages.json

```diff
@@ -2486,9 +2486,11 @@
     "message": "OpenSea is the first provider for this feature. More providers coming soon!",
     "description": "Description of a notification in the 'See What's New' popup. Describes Opensea Security Provider feature."
   },
+
   "notifications18Title": {
     "message": "Stay safe with security alerts"
   },
+
   "notifications19ActionText": {
     "message": "Enable NFT autodetection"
   },
```

Comment:
Changes to this file shouldn't be included in this PR. Please revert the changes